server/index.js

```diff
@@ -0,0 +1,14 @@
+import app from './src/configs/server';
+
+const appInstance = app();
+
+appInstance.create();
+appInstance.start();
```

Comment:
within start you can create the app instance, because start is highly coupled with create.

Reply:
Do the [new changes](https://github.com/pesto-students/batch-10-tripen/blob/30306f1c1ce782fa90c2d5cd7a39abcd8544831a/server/index.js#L5) sufficiently address this issue?